lib/parse/blueprint.js

```diff
@@ -26,6 +26,29 @@ module.exports = function(filePath, autoOptions, routeMap) {
                 addOptionsRoutesWhereMissing(allRoutesList);
             }
 
+            /**
+             * After populating the routeMap that is used to match incoming requests, 
+             * check API Blueprint for SERVICE name in the metadata. If one exists, use the value to 
+             * namespace the top level keys in routeMap. Note: these keys represent a url path to a resource
+             * and are compared against the path of incoming request object  in the route-handler middleware. 
+             */
+
+            var metadata = {};
+            result.ast.metadata.forEach(function(item){
+                metadata[item.name] = item.value;
+            });
+            var service = metadata.SERVICE;
```

Comment:
Could this possibly be set as a configuration that is passed in? So that we can turn this feature on or off?